Resources/Private/Fusion/Prototypes/Checkbox.Mutiple.fusion

```diff
@@ -0,0 +1,13 @@
+prototype(Neos.Fusion.Form:Checkbox.Multiple)  < prototype(Neos.Fusion.Form:Field) {
+    attributes = Neos.Fusion:DataStructure
+    value = ''
+    renderer = afx`
+        <input
+            {...props.attributes}
+            type="checkbox"
+            name={field.name}
+            value={props.value}
+            checked={(field.value && (Array.indexOf(field.value, this.value) > -1)) ? true : false}
```

Comment:
Nice change :) 

However I think for the "average integrator" it's hard to grasp the difference between props.value and field.value; as you need to specify both...

Reply:
Aaaah. I think if we use the eel Syntax you proposed with

```
prototype(Neos.Fusion.Form:Field)  < prototype(Neos.Fusion:Component) {
    @context.field = ${Form.fieldDefinition(form, this.name, this.property, this.value)}
    name = null
    property = null
    value = null
}

prototype(Neos.Fusion.Form:Form)  < prototype(Neos.Fusion:Component) {
    @context.form = ${Form.formDefinition(this.request, this.name, this.object, this.fieldNamePrefix)}
    request = ${request}
    name =  null
    object = null
    fieldNamePrefix = null
}
```

We won't have this problem, and he can just set "value" and that's it. Right?

Reply:
Yes i think getting rid of field.value will make it way easier to grasp